nncf/experimental/torch/sparsity/movement/structured_mask_strategy.py

```diff
@@ -32,7 +32,7 @@ def detect_supported_model_family(model: NNCFNetwork) -> Optional[str]:
     :return: The string name of the model family that `model` belongs to. If `model` is not supported,
         then returns None.
     """
-    model_pymodules = inspect.getmodule(model.get_nncf_wrapped_model()).__name__.split('.')
```

Comment:
Now the compressed model object and the original object are one and the same object, so there is no need for additional calls here and below when we need to query the original model for something.